src/main/resources/application.yaml

```diff
@@ -222,3 +222,6 @@ bail:
 flags:
   hmc-to-hearings-api:
     enabled: ${HMC_HEARINGS_LISTENING_ENABLED:true}
+
+pactbroker:
```

Comment:
is this variable being used in this changes? can't seem to find the reference point